### PR TITLE
[RFC / SQONE-697] update swiper version

### PR DESCRIPTION
**Pull request by GeoffDusome**
## What does this do/fix?

Updates Swiper to the most recent version including the following changes:
- Update Swiper styles to newest version, remove prefixed calls (since they get prefixed by us anyway), as well as update the variables file to include some new variables.
- Update `swiper` import call in `components/slider/js/slider.js`
- Update `swiper-container` class in `components/slider/Slider_Controller.php` to `swiper`

As far as I can tell from my testing of using the slider component, there are no breaking bugs in our implementation. Likely though, there may be some issues if this was implemented into an older site.

## QA

Links to relevant issues
- [SQONE-697](https://moderntribe.atlassian.net/browse/SQONE-697)

## Tests

Does this have tests?

- [ ] Yes
- [x] No, this doesn't need tests because there currently are no tests for the slider component.
- [ ] No, I need help figuring out how to write the tests.

### CODE REVIEW

**Review by LayaTaal — COMMENTED** [review]
@GeoffDusome I'm so glad you went through with this update! Referencing the SwiperJS docs was so hard being 2 major versions behind.

I played with updating this briefly, and I remember running into some breaking changes. I think that may be because I was changing from importing `SwiperCore` to `Swiper`, but I can't quite recall. It sounds like you have already checked, but wanted to call it out just in case.

File: wp-content/themes/core/integrations/swiper/css/_variables.pcss [hunk 1]
```diff
@@ -3,5 +3,13 @@
 	--swiper-navigation-size: 44px;
 	--swiper-navigation-color: var(--swiper-theme-color);
 	--swiper-pagination-color: var(--swiper-theme-color);
+	--swiper-pagination-bullet-size: 8px;
+	--swiper-pagination-bullet-width: 8px;
+	--swiper-pagination-bullet-height: 8px;
+	--swiper-pagination-bullet-inactive-color: #000;
```
[the remark this answers is not in the document]
Comment (GeoffDusome) [reply wp-content/themes/core/integrations/swiper/css/_variables.pcss]: Makes sense, I've made this change. I went ahead and used `--color-primary` for the `--swiper-theme-color` as well.

**Comment by GeoffDusome** [comment]
I went through the migration docs and couldn't find anything in our default implementation (`components/slider/js/slider.js`) that would be anything that breaks Swiper.

There is room to reduce the package size with the [modules installation](https://swiperjs.com/migration-guide#modules-installation) changes in `7.x` but those aren't necessary as they are backwards compatible with how we are currently using Swiper modules.

File: changelog.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ All notable changes to this project will be documented in this file.
 * Fixed: Removed return statement for an `add_action` in Admin_Subscriber.php.
 * Fixed: Branches that have Jira projects in them that contained numbers in the name would not properly prefix git commits, e.g. `feature/DATA22-3/some-new-feature`
 * Fixed: False positive on `have_posts()` using PHPStan if proceeded by an `if ( have_posts() ):`, utilizing a new [wordpress-overrides.stub file](dev/stubs/wordpress-overrides.stub)  
+* Bumped: Swiper to version `8.3.2`.
```
Comment (dpellenwood) [inline changelog.md:38]: Silly, but for consistency: 
```suggestion
* Updated: Swiper to version `8.3.2`.
```

**Review by dpellenwood — APPROVED** [review]
Love this. Thank you for tackling it, Geoff.

**Review by jamiepaul — APPROVED** [review]
This is great @GeoffDusome, thanks for updating!

**Review by Vinsanity — APPROVED** [review]
Nice work Geoff! Thank you.

**Review by LayaTaal — APPROVED** [review]
:rocket:

**Pull request merged**